src/main/java/org/reactome/release/dataexport/DataExporterStep.java

```diff
@@ -47,27 +47,28 @@
 	public void executeStep(Properties props) throws IOException {
 		logger.info("Beginning NCBI, UCSC, and Europe PMC export step...");
 
-		int version = Integer.parseInt(props.getProperty("reactomeVersion"));
+		int reactomeVersionNumber = Integer.parseInt(props.getProperty("reactomeNumber"));
```

Comment:
I do apologize for being a pain, but the configuration value is actually `releaseNumber`

Reply:
No problem - it was my mistake.

Done:
(main changes) https://github.com/reactome/release-data-exporter/pull/4/commits/ce8fc40feb3e45e062570f80ac9418416b0a6be7
(additional change that was missed) https://github.com/reactome/release-data-exporter/pull/4/commits/cbe885568e7d6e88327d351ee57d0ea74f92ed5b